pydocstringformatter/_configuration/toml_parsing.py

```diff
@@ -41,6 +44,22 @@ def parse_toml_option(
         except KeyError:
             raise UnrecognizedOption(f"Don't recognize option {opt}") from exc
 
+    if isinstance(action, BooleanOptionalAction):
+        option_strings = action.option_strings
+
+        if opt.startswith("no") and f"--{opt[3:]}" in option_strings:
+            opposite_opt = opt[3:]
+            val, opp_val = ["false", "true"][value], ["true", "false"][value]
```

Comment:
Can I break this by doing `no-numpydoc-section-hyphen-length = 1`? or `no-numpydoc-section-hyphen-length = yes`?

Reply:
```toml
no-numpydoc-section-hyphen-length = 1  # Pretty sure this would work fine
no-numpydoc-section-hyphen-length = "yes"  # Yes, I think it would break
```

could be sturdier by changing `value` to `bool(value)` in L52

Reply:
I'd prefer only accepting `true` or `false`. `bool("no")` would be `True` which is confusing imo.

Reply:
ok, i can see that, nontheless, that check would need to happen in L49, since by the time the program is in L52, it has already rejected the option and its only making an error message

what about ....
```python
# L49
if not isinstance(value, bool):
    raise ValueError(f"'{value}' is not a supported argument for '{opt}', please use either 'true' or 'false'")
# L50
```

Reply:
Perfect!

Reply:
Done